mesh/mesh.hpp

```diff
@@ -1308,6 +1316,69 @@ inline void Mesh::Rotate3(int &a, int &b, int &c)
    }
 }
 
+inline int Mesh::NodesOfGmshElement(int eleType)
+{
+   int nodes_of_gmsh_element[] =
```

Comment:
I'm not sure we can trust the compiler to to not generate this array and put it on the stack every time we want to use it.  It's probably best to use a static int array here defined in the class and initialized later.

Reply:
Sounds good

Reply:
This is done!